scripts/install-envoy-validate-plugin.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -euo pipefail
+
+source scripts/executor-info.sh
+
+PLUGIN_VERSION=$1
+
+if [ ! -f ".bin/protoc-gen-validate" ] || .bin/protoc-gen-validate --version | grep -q "$PLUGIN_VERSION"; then
```

Comment:
Guess you are missing a bracket here?
```suggestion
if [ ! -f ".bin/protoc-gen-validate" ] || .bin/protoc-gen-validate --version | grep -q "$PLUGIN_VERSION" ]; then
```

Reply:
I've gone and reworked the way I do this check to use a different method